gateways.json

```diff
@@ -1,43 +1,42 @@
 [
-	"https://ipfs.io/ipfs/:hash",
-	"https://gateway.ipfs.io/ipfs/:hash",
-	"https://ipfs.infura.io/ipfs/:hash",
-	"https://rx14.co.uk/ipfs/:hash",
-	"https://ninetailed.ninja/ipfs/:hash",
-	"https://upload.global/ipfs/:hash",
+	"https://ipfs.io/:protocol/:hash",
+	"https://gateway.ipfs.io/:protocol/:hash",
+	"https://ipfs.infura.io/:protocol/:hash",
+	"https://rx14.co.uk/:protocol/:hash",
+	"https://ninetailed.ninja/:protocol/:hash",
+	"https://upload.global/:protocol/:hash",
 	"https://ipfs.globalupload.io/:hash",
```

Comment:
How to deal with this case? It'll test ipns and ipfs even if ipns should not work?

Reply:
Yes, for now that would happen.